reactVR_course/vr/client.js

```diff
@@ -31,12 +30,12 @@ function init(bundle, parent, options) {
     cursorVisibility: 'visible', */
     enableHotReload: true,
     ...options,
-    // Register dom overlay module upon initialization.
-    nativeModules: [domOverlayModule],
+    // TODO: Comment in to enable Native Module
```

Comment:
Kan endre til "uncomment" istedenfor "comment in", men det er veeeldig pirk :)

Reply:
Tar den i samme slengen ;)